CODE_OF_CONDUCT.md

```diff
@@ -1,7 +1,8 @@
 # Guidelines for Respectful Communication
 
 As members of a Haskell Foundation affiliated project, we commit ourselves to
-a high standard of public behaviour. We have one over-arching rule:
+a high standard of public behaviour and we expect the same of participants on
```

Comment:
Our public-facing language of choice is US English, right, so that should be
```suggestion
a high standard of public behavior and we expect the same of participants on
```

I see a few more down, but I can't edit them in the review process. Shall I do a force-push?

Reply:
Or can't we even change the spelling of the text further down :-)

Reply:
No, I'd rather keep the original text as much as possible.